backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepository.java

```diff
@@ -62,4 +64,12 @@ public UUID createShop(CreateShopRequest createShopRequest) {
 	public Map<UUID, Shop> getAllShops() {
 		return shopMap;
 	}
+
+	public List<Token> getTokensByShopId(UUID shopId) {
```

Comment:
We can name this "getTokens".
Since shopId is being specified in the params, it is implicit and so we don't need the method name as "getTokensByShopId".

Reply:
done!